[2022-09-18 18:43:21,225] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 18:43:21,305] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 18:43:21,309] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:43:21,311] {taskinstance.py:1018} INFO - Starting attempt 2 of 3
[2022-09-18 18:43:21,320] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:43:21,441] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-12T03:00:00+00:00
[2022-09-18 18:43:21,456] {standard_task_runner.py:51} INFO - Started process 3038 to run task
[2022-09-18 18:43:21,512] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-12T03:00:00+00:00', '--job-id', '332', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmpg7py9pq9']
[2022-09-18 18:43:21,523] {standard_task_runner.py:76} INFO - Job 332: Subtask write_query_big_query_table
[2022-09-18 18:43:21,872] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 18:43:22,135] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-12T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-12T03:00:00+00:00
[2022-09-18 18:43:22,150] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 18:43:22,256] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 18:43:22,387] {bigquery.py:1510} INFO - Inserting job airflow_1663526602381999_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 18:43:23,876] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663526602381999_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663526602381999_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 18:43:23,939] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220912T030000, start_date=20220918T184321, end_date=20220918T184323
[2022-09-18 18:43:24,196] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 19:41:44,059] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 19:41:44,327] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 19:41:44,331] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:41:44,333] {taskinstance.py:1018} INFO - Starting attempt 2 of 3
[2022-09-18 19:41:44,335] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:41:44,504] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-12T03:00:00+00:00
[2022-09-18 19:41:44,552] {standard_task_runner.py:51} INFO - Started process 3874 to run task
[2022-09-18 19:41:44,620] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-12T03:00:00+00:00', '--job-id', '431', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmph6h4btd_']
[2022-09-18 19:41:44,667] {standard_task_runner.py:76} INFO - Job 431: Subtask write_query_big_query_table
[2022-09-18 19:41:45,078] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 19:41:45,429] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-12T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-12T03:00:00+00:00
[2022-09-18 19:41:45,465] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 19:41:45,578] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 19:41:45,774] {bigquery.py:1510} INFO - Inserting job airflow_1663530105770682_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 19:41:47,746] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663530105770682_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663530105770682_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 19:41:47,842] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220912T030000, start_date=20220918T194144, end_date=20220918T194147
[2022-09-18 19:41:48,320] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 20:19:02,275] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 20:19:02,334] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [queued]>
[2022-09-18 20:19:02,336] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 20:19:02,338] {taskinstance.py:1018} INFO - Starting attempt 2 of 3
[2022-09-18 20:19:02,339] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 20:19:02,392] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-12T03:00:00+00:00
[2022-09-18 20:19:02,404] {standard_task_runner.py:51} INFO - Started process 3311 to run task
[2022-09-18 20:19:02,440] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-12T03:00:00+00:00', '--job-id', '485', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmpp55f_whk']
[2022-09-18 20:19:02,450] {standard_task_runner.py:76} INFO - Job 485: Subtask write_query_big_query_table
[2022-09-18 20:19:02,605] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-12T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 20:19:02,819] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-12T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-12T03:00:00+00:00
[2022-09-18 20:19:02,841] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 20:19:02,952] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 20:19:03,071] {bigquery.py:1510} INFO - Inserting job airflow_1663532343063761_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 20:19:04,250] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663532343063761_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663532343063761_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 20:19:04,277] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220912T030000, start_date=20220918T201902, end_date=20220918T201904
[2022-09-18 20:19:04,414] {local_task_job.py:118} INFO - Task exited with return code 1
